src/content/docs/connections/concurrency.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Concurrency"
+---
+
+Kùzu implements MVCC ([Multiversion Concurrency Control](https://en.wikipedia.org/wiki/Multiversion_concurrency_control))
```

Comment:
I wonder if we need to mention about MVCC here. Although MVCC is related to concurreny, the problem discussed in this section is not specific to MVCC. But I'll let @semihsalihoglu-uw and @ray6080 decide.

Reply:
Sure, but I'm not suggesting anywhere that the problem is related to MVCC - I put it in there because it's a term that database users might know, and it's helpful to mention that we already implement it. The issues listed below are unrelated to this fact, which I was hoping is clear.